src/main.jsx

```diff
@@ -1,13 +1,23 @@
 import React from 'react';
 import ReactDOM from 'react-dom/client';
+import {
+  useQuery,
+  useMutation,
+  useQueryClient,
+  QueryClient,
+  QueryClientProvider,
+} from '@tanstack/react-query';
+import { ReactQueryDevtools } from '@tanstack/react-query-devtools';
 import { BrowserRouter } from 'react-router-dom';
 import App from './App.jsx';
 import './index.scss';
 
+const queryClient = new QueryClient();
+
 ReactDOM.createRoot(document.getElementById('root')).render(
-  <React.StrictMode>
```

Comment:
Why did you remove `StrictMode` ?

Reply:
In the video tutorial on React Query, some guy deleted it to remove repeated renderings. As far as I understand, it was not worth deleting strict mode..